<ui version="4.0" >
 <class>ControlPanel</class>
 <widget class="QWidget" name="ControlPanel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>63</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>1</number>
   </property>
   <property name="topMargin" >
    <number>1</number>
   </property>
   <property name="rightMargin" >
    <number>1</number>
   </property>
   <property name="bottomMargin" >
    <number>1</number>
   </property>
   <property name="horizontalSpacing" >
    <number>0</number>
   </property>
   <property name="verticalSpacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="prevButton" >
       <property name="minimumSize" >
        <size>
         <width>42</width>
         <height>28</height>
        </size>
       </property>
       <property name="whatsThis" >
        <string>Skip to previous song</string>
       </property>
       <property name="shortcut" >
        <string>Ctrl+PgUp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="playButton" >
       <property name="minimumSize" >
        <size>
         <width>42</width>
         <height>28</height>
        </size>
       </property>
       <property name="whatsThis" >
        <string>Play</string>
       </property>
       <property name="shortcut" >
        <string>Ctrl+Ins</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pauseButton" >
       <property name="minimumSize" >
        <size>
         <width>42</width>
         <height>28</height>
        </size>
       </property>
       <property name="whatsThis" >
        <string>Pause</string>
       </property>
       <property name="shortcut" >
        <string>Ctrl+Home</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="stopButton" >
       <property name="minimumSize" >
        <size>
         <width>42</width>
         <height>28</height>
        </size>
       </property>
       <property name="whatsThis" >
        <string>Stop</string>
       </property>
       <property name="shortcut" >
        <string>Ctrl+End</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="nextButton" >
       <property name="minimumSize" >
        <size>
         <width>42</width>
         <height>28</height>
        </size>
       </property>
       <property name="whatsThis" >
        <string>Skip to next song</string>
       </property>
       <property name="shortcut" >
        <string>Ctrl+PgDown</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="10" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="volumeLabel" />
     </item>
     <item>
      <widget class="QSlider" name="volumeSlider" >
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="timeLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TimeLabel" name="elapsedLabel" />
     </item>
     <item>
      <widget class="TimeSlider" name="timeSlider" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TotalTimeLabel" name="totalLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item rowspan="2" row="0" column="9" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="10" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>108</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" colspan="5" >
    <widget class="QLabel" name="titleLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth" >
      <number>1</number>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="0" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" colspan="5" >
    <widget class="QLabel" name="artistLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>211</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="0" column="8" >
    <widget class="QPushButton" name="coverArtButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="0" column="7" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TotalTimeLabel</class>
   <extends>QLabel</extends>
   <header>timelabel.h</header>
  </customwidget>
  <customwidget>
   <class>TimeSlider</class>
   <extends>QSlider</extends>
   <header>timeslider.h</header>
  </customwidget>
  <customwidget>
   <class>TimeLabel</class>
   <extends>QLabel</extends>
   <header>timelabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>prevButton</tabstop>
  <tabstop>playButton</tabstop>
  <tabstop>pauseButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>nextButton</tabstop>
  <tabstop>coverArtButton</tabstop>
  <tabstop>volumeSlider</tabstop>
  <tabstop>timeSlider</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
